src/processing/script/index.js

```diff
@@ -164,7 +165,7 @@ export function isScriptProcessed (code) {
     return PROCESSED_SCRIPT_RE.test(code);
 }
 
-export function processScript (src, withHeader) {
+export function processScript (src, withHeader, wrapLastExprViaProcessHtml) {
```

Comment:
`processScript (src, { withHeader, wrapLastExprViaProcessHtml })`
two parameters: first - `src`, second - processing options

`wrapLastExprViaProcessHtml` -> `wrapLastExprWithProcessHtml`

Reply:
The `processScript` function is used in testcafe.

Reply:
Ok. Don't change this. It is used in `testcafe-legacy-api` too